st2common/st2common/openapi.yaml

```diff
@@ -977,6 +977,11 @@ paths:
           description: Show secrets in plain text
           type: boolean
       x-parameters:
+        - name: filter
+          in: query
+          x-as: advanced_filters
```

Comment:
Could this potentially have negative security implications?

Like injections or ability to retrieve data which shouldn't be allowed. Do we need a whitelist of keys and operators like we have for regular filters?

Reply:
I could not think of anything that might be a security issue for execution document. An intruder may want to try and brute force the secrets, but I would not consider it a massive threat at the moment.

If you have a whitelist in mind, lmk.

We could also hide this feature behind rbac, although I still not believe it's dangerous enough.